app/src/main/java/com/gcect/gcectapp/viewmodels/PatentViewmodel.kt

```diff
@@ -0,0 +1,22 @@
+package ViewModel
```

Comment:
bro always recheck the package name before PR. coz this is simply crashing the application.